lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/openturns/FunctionalChaosAlgorithm.hxx

```diff
@@ -141,6 +153,8 @@ private:
   /** Maximum residual */
   Scalar maximumResidual_;
 
```

Comment:
It seems to me that if the `FunctionalChaosAlgorithm` has a new constructor input argument `weights`, then a new corresponding attribute `weights_` should be defined at this point, isn'it? Then children classes would use it, as they should.

```suggestion
    /** Weights */
    Point weights_;
```

@sofianehaddad has a similar suggestion I think.